saleor/dashboard/order/utils.py

```diff
@@ -29,8 +29,12 @@ def _create_pdf(rendered_template, absolute_url):
 def create_invoice_pdf(group_pk, absolute_url):
     group = (DeliveryGroup.objects.prefetch_related(
         'items', 'order', 'order__user', 'order__shipping_address',
```

Comment:
If you prefetch `some_field__some_fields_field` then `some_field` will be automatically prefetched, no need to add it to the list.